### ISSUE TITLE
Incorrect element nesting

### ISSUE THREAD

**Issue by benhammondmusic** [SatcherInstitute/health-equity-tracker#2836]
issues on:
- http://localhost:3000/new-methodology/topics/covid#covid-19 
- http://localhost:3000/new-methodology/topics/pdoh#pdoh
- http://localhost:3000/new-methodology/definitions/metrics

view console in dev tools similar to below (sometimes other elements error)

```

Warning: validateDOMNesting(...): <ol> cannot appear as a descendant of <p>.
    at ol
    at p
    at article
    at section
    at Covid19Link

```
